css/index.css

```diff
@@ -1,124 +1,221 @@
-/*   NAVBAR STYLING  */
```

Comment:
Regarding `Keri-portrait.png.zip`: I'm guessing this is an original full resolution you want to keep? I would generally avoid zipping images and other binary files -- it makes them harder to work with and often doesn't save much storage. You can look at tools for PNG compression, though.